app/dashboard/src/App.tsx

```diff
@@ -122,14 +124,14 @@ LocalStorage.registerKey('inputBindings', {
       ),
 })
 
-// ======================
-// === getMainPageUrl ===
-// ======================
+// ==================
+// === getRootUrl ===
+// ==================
 
-/** Returns the URL to the main page. This is the current URL, with the current route removed. */
-function getMainPageUrl() {
+/** Returns the URL to the app root. This is the current URL, with the current route removed. */
+function getRootUrl() {
   const mainPageUrl = new URL(window.location.href)
-  mainPageUrl.pathname = mainPageUrl.pathname.replace(appUtils.ALL_PATHS_REGEX, '')
+  mainPageUrl.pathname = appBaseUrl.APP_BASE_URL
```

Comment:
Is this behaviour change intentional? It looks like `APP_BASE_URL` is now only computed once whereas before it would be computed every time this function was called?

Reply:
i think it's safe to assume the base URL at which the static app is hosted doesn't change over time :p

Reply:
...fair